pyshield/_config.py

```diff
@@ -1,60 +1,1711 @@
 import dataclasses
+import math
 from enum import Enum, unique
 from typing import List, Optional, Tuple
 
 import f90nml
 
+import ndsl.constants as constants
+import ndsl.namelist as nml
+import pyshield.constants as physcons
 from ndsl import MetaEnumStr
-from ndsl.namelist import Namelist, NamelistDefaults
 
 
 DEFAULT_INT = 0
+DEFAULT_FLOAT = 0.0
+DEFAULT_STR = ""
 DEFAULT_BOOL = False
 DEFAULT_SCHEMES = ["GFS_microphysics"]
 
 
 @unique
 class PHYSICS_PACKAGES(Enum, metaclass=MetaEnumStr):
     GFS_microphysics = "GFS_microphysics"
+    SHiELD_microphysics = "SHiELD_microphysics"
+
+
+# Global set of physics namelist defaults
+# attached to class for namespacing and static typing
+class NamelistDefaults:
+    tau_r2g = 900.0  # rain freezing during fast_sat
```

Comment:
None of those comments will parse in the auto_docs.

```suggestion
    tau_r2g = 900.0
    """rain freezing during fast_sat"""
```